CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+- [Pre-built images](#pre-built-images)
+- [Build Requirements](#build-requirements)
+- [Building Executables and Images from Scratch](#building-executables-and-images-from-scratch)
+- [Development](#development)
+- [Local Testing](#local-testing)
+
+## Pre-built images
+
+For a list of supported public images of Tornjak along with usage instructions please see our [USAGE document](./USAGE.md).
+
+## Build Requirements
+
+In order to build from scratch, we require the following installations:
+- [Docker]() for the backend
+- [npm]() for the frontend
+
```

Comment:
Don't we need a GoLang installed to compile the Backend?

Reply:
the executables are built in a container - so the container does the compilation and contains golang already, from what i understand